app/build.gradle.kts

```diff
@@ -34,6 +69,7 @@ android {
 			// Use different application id to run release and debug at the same time
 			applicationIdSuffix = ".debug"
 			resValue("string", "app_name", "@string/app_name_debug")
+			versionNameSuffix =  "-debug-" + getBranchName() + "-" + getGitHash() + (if (isTreeClean()) "-treeClean" else "-treeDirty")
```

Comment:
I usually prefer to hide this information somewhere else so the name looks less awkward. It looks a lot better when you run the debug build more often than the production one.

Reply:
Hmmm, I get that, but how often does one really look at the version string?

Reply:
Ah, this is the version string. I have seen it added to the app name in the past, which is what I thought was happening here.